ui/ducks/send/send.js

```diff
@@ -1908,20 +1923,35 @@ export function updateRecipientUserInput(userInput) {
         }
       }
     }
-
     return new Promise((resolve) => {
-      debouncedValidateRecipientUserInput(
-        dispatch,
-        {
-          userInput,
-          chainId,
-          tokens,
-          useTokenDetection,
-          tokenAddressList,
-          isProbablyAnAssetContract,
-        },
-        resolve,
-      );
+      try {
+        debouncedValidateRecipientUserInput(
+          dispatch,
+          {
+            userInput,
+            chainId,
+            tokens,
+            useTokenDetection,
+            tokenAddressList,
+            isProbablyAnAssetContract,
+            udTlds: state.UNS.tlds,
```

Comment:
I assume this try/catch is because state.UNS.tlds could be undefined? this can be simplified to:

state.UNS?.tlds, which will then just pass 'udTlds' to undefined and avoids the try/catch.